xbmc/cores/RetroPlayer/rendering/RPRenderManager.h

```diff
@@ -175,6 +177,7 @@ namespace RETRO
     bool m_bHasCachedFrame = false;
     std::set<std::string> m_failedShaderPresets;
     bool m_bTriggerUpdateResolution = false;
+    std::atomic<bool> m_bFlush;
```

Comment:
If my c++11 is correct, this can be initialized in the header like `std::atomic<bool> m_bFlush{};`